AdminUi/src/AdminUi/ClientApp/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:20
+
+RUN echo "RUN echo"
+
+WORKDIR /app
+
+COPY AdminUi/src/AdminUi/ClientApp/package*.json ./
+COPY AdminUi/src/AdminUi/ClientApp/tsconfig.json ./
+COPY AdminUi/src/AdminUi/ClientApp/tsconfig.app.json ./
+
+RUN npm install
+
+RUN npm install -g @angular/cli
```

Comment:
Why is this separated from the standard npm install call?

Reply:
It is not needed, true. I have removed it.